CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Fix memory leak in the global `MeterProvider` when identical instruments are repeatedly created. (#5754)
 - Fix panic instruments creation when setting meter provider. (#5758)
+- Fix metric config WithResource function and LoggerProvider#newProviderConfig to merge provided resource with resource information from the environment. (#5773)
```

Comment:
This would be more readable as two changelog entries IMHO. Also, this technically isn't a bugfix, but a new feature.

Reply:
It should be "Added", as it's a new feature.

Reply:
That's what I did, so we should be good here. 👍